src/member/fixtures.ts

```diff
@@ -0,0 +1,48 @@
+import { MemberStorageItemsResponse } from '@graasp/sdk';
+
+import { PaginationParams } from '../types.js';
+
+export const itemsResponse: MemberStorageItemsResponse = {
+  items: [
```

Comment:
This is not correct with respect to the type you are using in the API function. It will return it on `data` not `items`.

Reply:
So here, i'll change it in the sdk before